.yarn/patches/@metamask-selected-network-controller-npm-18.0.0-2e967a6d5c.patch

```diff
@@ -0,0 +1,120 @@
+diff --git a/dist/chunk-D745I7P5.mjs b/dist/chunk-D745I7P5.mjs
```

Comment:
This patches https://github.com/MetaMask/core/pull/4679, and I will try to cut a release for it